lib/platform/gitlab/index.spec.ts

```diff
@@ -44,6 +49,24 @@ describe('platform/gitlab', () => {
     httpMock.reset();
   });
 
+  async function initFakePlatform(version: string) {
+    httpMock.scope(gitlabApiHost).get('/api/v4/user').reply(200, {
+      email: 'a@b.com',
+      name: 'Renovate Bot',
+    });
+    httpMock
+      .scope(gitlabApiHost)
```

Comment:
```suggestion
    })
```

reuse same scopes as in other tests

Reply:
Good point, thanks!